tmt/convert.py

```diff
@@ -551,8 +563,8 @@ def read_nitrate_case(testcase, makefile_data=None):
                         data['tier'] = num
                 else:
                     data['tier'] = num
-
-        data['tag'] = sorted(tags)
+        # Include possible multihost tag
+        data['tag'] = sorted(tags) + data.get('tag', [])
```

Comment:
I think you want to do `sorted(tags + data.get('tag', []))` to preserve old behaviour with sorted tags. in your case multihost tag will had special handling and will be everytime as last.